ipld/src/lib.rs

```diff
@@ -44,7 +117,10 @@ pub fn from_ipld<T>(value: &Ipld) -> Result<T, String>
 where
     T: DeserializeOwned,
 {
-    // TODO find a way to convert without going through byte buffer
+    // TODO update to not go through byte buffer to convert
```

Comment:
This is so janky its funny. But agree we dont need to fix this yet haha

Reply:
I mean, it's not THAT janky, not like it's a common concept to be able to go from a generalized type to a specific one without a buffer usually. This is correct, just not efficient

Reply:
Definitely correct, not questioning. I just think its funny how we have to go to CBOR first then deserialize. Anyways, nothing needs to be fixed here.